docs/DIFF_STUDY_CLINICAL.md

```diff
@@ -0,0 +1,106 @@
+# Compare current versus build
+This documentation addresses a [QC script](../scripts/diff_studies.py) for clinical metadata. It streamlines the process of identifying and summarizing changes slated to be made
+
+```sh
+python3 scripts/diff_studies.py --help
+usage: diff_studies.py [-h] [-u URL] [-s STUDY] [-t TOKEN] [-d DATA_DIR]
+
+Compare local clinical data to server
+
+options:
+  -h, --help            show this help message and exit
+  -u URL, --url URL     url to search against
+  -s STUDY, --study STUDY
+                        Cancer study ID to compare on server
+  -t TOKEN, --token TOKEN
+                        Token file obtained from Web API
+  -d DATA_DIR, --datasheet-dir DATA_DIR
+                        Directory containing data_clinical_*.txt
+```
+
+## INPUTS:
+ - `-u, --url`: cBioportal api deployment site. Default: https://pedcbioportal.kidsfirstdrc.org/api/v2/api-docs
+ - `-s, --study`: cBioportal cancer study ID, i.e. `pbta_all`
+ - `-t, --token`: File obtained from navigating to https://pedcbioportal.kidsfirstdrc.org/webAPI#using-data-access-tokens, then clicking on `Download Token`. File is reusable
+ - `-d, --datasheet-dir`: Name of directory containing `data_clinical_patient.txt` and `data_clinical_sample.txt` being vetted for upload
+
+## OUTPUTS:
+Essentially two change log files, `patient_portal_v_build.txt` and `sample_portal_v_build.txt`.
+For the patient and sample views, each file respectively has:
+ - A list, one per line, per ID, per attribute, of what would change if the data were loaded
+ - A list of IDs that would be removed from the portal, if any
+ - A list of IDs that would be added in any
```

Comment:
I'm assuming this is supposed to be:
- A list of IDs that would be added, if any

Reply:
yup!